moveit_ros/moveit_servo/demos/cpp_interface/demo_joint_jog.cpp

```diff
@@ -102,7 +113,13 @@ int main(int argc, char* argv[])
     }
     else if (status != StatusCode::INVALID)
     {
-      trajectory_outgoing_cmd_pub->publish(composeTrajectoryMessage(servo_params, joint_state));
+      updateSlidingWindow(joint_state, joint_cmd_rolling_window, servo_params.max_expected_latency);
+      if (auto msg = composeTrajectoryMessage(servo_params, joint_cmd_rolling_window))
```

Comment:
Can this be `const`?

Reply:
Are yo referring to `msg`?

Reply:
Yeah, sorry -- and the "same" is for the twist/pose demo files too.